fetch.bs

```diff
@@ -5926,6 +6219,9 @@ the associated steps:
  <li><p>Let <var>steps</var> be to return the result of <a>package data</a> with the first argument
  given, <var>type</var>, and <var>object</var>'s <a for=Body>MIME type</a>.
 
+ <li><p><a>Upon fulfillment</a> of <var>promise</var>, call <var>object</var>'s
+ <a for=Body>on body consumed</a>.
```

Comment:
One thing this setup means is that unless you consume you don't get a RT object. Is that the case? Also, it's a bit weird that above we do care about errors, but here we don't.